frontend/src/i18n/messages/en/index.ts

```diff
@@ -295,6 +294,69 @@ export const messages = {
         },
       ],
     },
+    progress: {
+      title: "Our progress so far",
+      list: [
+        {
+          title: "RESTful API",
+          content:
+            "Our new modern API makes grants opportunity data more accessible, with an API‑first approach that prioritizes data and ensures that the Grants.gov website, 3rd‑party apps, and other services can more easily access grants data.",
+        },
+        {
+          title: "Code Challenge pilot",
+          content:
+            "We’re excited to announce the successful pilot of the Collaborative Code & Design Challenge, which laid the groundwork for a scalable framework to support future open-source contributions. This event was conducted in a fully remote environment to bring together participants who engaged in innovative problem-solving and collaboration.<p><linkWikiInvolved>Get involved in our open-source community</linkWikiInvolved></p>",
+        },
+        {
+          title: "Search interface launch",
+          content:
+            "Simpler.Grants.gov now has improved search capabilities that make it easier to find funding opportunities published by Grants.gov.<p><linkSearch>Try the new simpler search</linkSearch></p>",
+        },
+
+        {
+          title: "Co-Design Group recruitment",
+          content:
+            "We've recruited a cohort of community members with lived experience using Grants.gov to participate in the design process. Through a long-term engagement, these co-designers will ensure what we build delivers the most value to grant seekers who struggle most with the grants experience.",
+        },
+        {
+          title: "Opportunity Listing",
+          content:
+            "You can now view opportunity details on Simpler.Grants.gov, with action-oriented information in the right column and detailed content on the left. With this new design, grant seekers can make faster, more informed decisions about opportunities.",
+        },
+        {
+          title: "Search UI usability testing",
+          content:
+            "We’ve conducted sessions with grant seekers, grantors, and HHS staff to test the new design. This study revealed findings and uncovered tangible issues to be resolved in the next Search UI iteration.",
+        },
+      ],
+    },
+    next: {
+      title: "What's happening next",
+      link: "View all of our deliverables on GitHub",
+      list: [
+        {
+          title: "Authenticate via Login.gov",
+          content:
+            "Enable grant seekers to create an account on Simpler.Grants.gov using the Login.gov single sign-on platform to minimize the steps and friction users experience when signing up.<p><linkGithub2640>Follow #2640 on GitHub</linkGithub2640></p>",
+        },
+        {
+          title: "Search & Opportunity Listing improvements",
```

Comment:
based on the Figma "Search & Opportunity Listing improvements" should go 3rd in the list, and "Full support for Opportunity Listing attachments" should be second